.travis.yml

```diff
@@ -3,15 +3,15 @@ cache:
   - cargo
 rust:
   - stable
-  - 1.23.0
+  - 1.31.0
```

Comment:
Since this increases the minimum required compiler version, I don't want to land it. Unless someone commits to working on something like slog-v3 or something.  

Formatting changes are fine with me.

Reply:
The wiki says [that bumping the minimum required Rust version is considered a minor-version change](https://github.com/slog-rs/slog/wiki/Compatibility-Policy-&-Rust-version#minimum-version-of-rust). I guess it'd be a little silly to bump the minor version if the only thing we're adding is Rust 2018 support.

Rust 1.23 is two and a half years old. At some point in the future (not necessarily now), I think it's reasonable to change it without a complete update to slog-v3.

Reply:
Yeah. At some point sure. :)

Reply:
We're switching to Github Actions, so this `.travis.yml` is irrelevant.

Also, you approved bumping the MSRV below so this problem is double-resolved.